apps/emqx_rule_engine/src/emqx_rule_engine.erl

```diff
@@ -478,6 +477,13 @@ with_parsed_rule(Params = #{id := RuleId, sql := Sql, actions := Actions}, Creat
                 conditions => emqx_rule_sqlparser:select_where(Select)
                 %% -- calculated fields end
             },
+            InputEnable = maps:get(enable, Params, true),
+            Enable =
+                case validate_bridge_existence_in_actions(Rule0) of
+                    ok -> InputEnable;
+                    {error, _} -> false
```

Comment:
this is something user needs to fix.
maybe write an error level log here

Reply:
That's what I meant by "silently disabling it" when I suggested this solution. :joy_cat: 
Ok, I'll add a log here.